types/stylelint/index.d.ts

```diff
@@ -161,6 +180,7 @@ declare namespace stylelint {
 		stylelintError?: boolean;
 		stylelintWarning?: boolean;
 		config?: Config;
+		lexer?: Lexer;
```

Comment:
@jeddy3 in response to https://github.com/stylelint/stylelint/pull/8297#pullrequestreview-2574034022

If it's very likely to happen, shouldn't we opt for
```suggestion
		lexers: {
		  csstree?: Lexer;
		}
```
instead?